public/app/percona/dbaas/components/DBCluster/DBCluster.types.ts

```diff
@@ -82,9 +82,28 @@ export interface DBClusterExpectedResources {
 }
 
 interface DBClusterResources {
-  cpu: number;
-  disk: number;
-  memory: number;
+  cpu: ResourcesWithUnits;
+  disk: ResourcesWithUnits;
+  memory: ResourcesWithUnits;
+}
+
+export interface ResourcesWithUnits {
+  value: number;
+  units: ResourcesUnits | CpuUnits;
+}
+
+export enum ResourcesUnits {
+  BYTES = 'Bytes',
+  KB = 'KB',
+  MB = 'MB',
+  GB = 'GB',
```

Comment:
Quick question, maybe @atymchuk can answer it: is there a reason why we're using GB at some places, like here, and GiB in others like Backups?

Reply:
If you mean the label maybe is just inconsistency (or not). `GiB` is used when you represent the value with powers of 2, `GB` is used when you represent the value with powers of 10. In DBaaS we are using powers of 10 (for simplicity and because Kubernetes handles it), hence the "GB".

Reply:
IMHO, we should be using GiB everywhere, hence base 2. We are in IT, so 2 makes more sense :)

Reply:
I would suggest [this](https://en.wikipedia.org/wiki/Gigabyte) read, it explain well the difference between the GB and GiB. 

TL;DR; - GiB make sense to be used for describing RAM or any other device that it's physical design is tight to power of 2. For example, a good use is if you want to be 100% precise when printing a value (1 GiB vs 1.074 GB) and don't want to look pedantic in the same place. At the end the amount that is being described is the same, just presented differently.

Reply:
@atymchuk k8s speaks whatever we want based on the notation we pass (see [here](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory)). We are using base 10, hence the `GB`, so cannot change the label here without changing the rest of the implementation (to avoid inconsistencies). But I agree that base 2 makes more sense for this things (e.g. memory as @browseman said), we just started with base 10 and went with it, it works fine, but definitely can be changed.

Reply:
Wow, I learned something. This is exactly why I love code reviews 🥇